Software/Python/easygopigo3.py

```diff
@@ -337,12 +337,14 @@ def spin_right(self):
         """
         Rotate the `GoPiGo3` towards the right while staying on the same spot.
 
+        This differs from the :py:meth:`~easygopigo3.EasyGoPiGo3.right` method as both wheels will be rotating but in different directions.
+        This causes the robot to spin in place, as if doing a pirouette.
+
         For setting the motor speed, use :py:meth:`~easygopigo3.EasyGoPiGo3.set_speed`.
         Default ``speed`` is set to **300** - see :py:meth:`~easygopigo3.EasyGoPiGo3.__init__`.
 
         .. important::
-             | This differs from the right() method as both wheels will be rotating but in different directions.
-             | This causes the robot to spin in place, as if doing a pirouette
+             This is a particularity of :py:meth:`~easygopigo3.EasyGoPiGo3.steer` method, thus you can achieve the same by calling ``steer(100, -100)``.
```

Comment:
why is steer mentioned in the documentation for spin_right?

Reply:
At the very least, rephrase it. 
"You can achieve the same effect by calling ``steer(100, -100)"

I wouldn't say the spin_right/left() methods are particuliarities of the steer() methods, since it doesn't call steer() . They kinda live as cousins, not as descendants.